capacity4more/modules/c4m/vocab/c4m_vocab_widget/templates/c4m_vocab_widget.tpl.php

```diff
@@ -4,6 +4,12 @@
  * Template to render the Vocabulary widget.
  */
 ?>
+<div ng-show="'<?php print $vocabulary_machine_name; ?>' == 'c4m_related_topic'">
```

Comment:
oh, sorry I misunderstood you suggestion you told me offline. Yes - please use the ``field_info_instance()`` to get the text